notebooks/classify/signature.py

```diff
@@ -0,0 +1,249 @@
+import numpy as np
+import pandas as pd
+import re
+import datetime
+import matplotlib.pyplot as plt
+import ipywidgets as widgets
+from typing import Dict, List
+
+class DataSignature:
```

Comment:
Class docstring
Might be helpful in the class to have an Examples section, showing how to use the class.

If you use numpy-style docstrings you can include this like
Examples
----------
>>> data_sig = DataSignature(table=data)
Call X to do Y
>>> data_sig.method()